README.md

```diff
@@ -1,250 +1,163 @@
+and how to install them.
+
+**Note**: For most hyervisor VT-x/AMD-v virtualization must be enabled in BIOS. For Hyper-V, however,
+it needs to be disabled.
 
+<a name="instructions"></a>
 ### Instructions
 
-* OS X
-  ```
-  brew cask install minishift
-  ```
+Download the binary matching your host OS from the Minishift [releases page](https://github.com/minishift/minishift/releases) and copy it to your preferred location. Make sure it is executable and optionally rename
+it to _minishift_.
 
-* Other OS
+On OS X you can also use [Homebrew Cask](https://caskroom.github.io) to install Minishift:
 
-  See the installation instructions for the [latest release](https://github.com/jimmidyson/minishift/releases).
+  ```
+  $ brew cask install minishift
+  ```
 
+<a name="quickstart"></a>
 ## Quickstart
 
-Here's a brief demo of minishift usage.
-If you want to change the VM driver add the appropriate `--vm-driver=xxx` flag to `minishift start`. Minishift Supports
-the following drivers:
+This section contains a brief demo of Minishift usage.
 
-* virtualbox
-* vmwarefusion
-* kvm ([driver installation](DRIVERS.md#kvm-driver))
-* xhyve ([driver installation](DRIVERS.md#xhyve-driver))
+<a name="starting-minishift"></a>
+### Starting Minishift
 
 Note that the IP below is dynamic and can change. It can be retrieved with `minishift ip`.
 
 ```shell
 $ minishift start
 Starting local OpenShift cluster...
-Running pre-create checks...
-Creating machine...
-Starting local OpenShift cluster...
-
-$ oc run hello-minishift --image=gcr.io/google_containers/echoserver:1.4 --port=8080 --expose --service-overrides='{"apiVersion": "v1", "spec": {"type": "NodePort"}}'
-service "hello-minishift" created
-deploymentconfig "hello-minishift" created
-
-# We have now launched an echoserver pod but we have to wait until the pod is up before curling/accessing it
-# via the exposed service.
-# To check whether the pod is up and running we can use the following:
-$ oc get pod
-NAME                              READY     STATUS              RESTARTS   AGE
-hello-minishift-3383150820-vctvh   1/1       ContainerCreating   0          3s
-
-# We can see that the pod is still being created from the ContainerCreating status
-$ oc get pod
-NAME                              READY     STATUS    RESTARTS   AGE
-hello-minishift-3383150820-vctvh   1/1       Running   0          13s
-
-# We can see that the pod is now Running and we will now be able to curl it:
-$ curl $(minishift service hello-minishift --url)
-CLIENT VALUES:
-client_address=192.168.99.1
-command=GET
-real path=/
 ...
+   OpenShift server started.
+   The server is accessible via web console at:
+       https://192.168.99.128:8443
 
-$ minishift stop
-Stopping local OpenShift cluster...
-Stopping "minishift"...
-```
+   You are logged in as:
+       User:     developer
+       Password: developer
 
-### Driver plugins
+   To login as administrator:
+       oc login -u system:admin
```

Comment:
Can we please add a line saying , user needs to download the `OC` binary manually. Otherwise it is confusing for users.